extension/src/helpers/stellar.ts

```diff
@@ -1,2 +1,2 @@
 export const truncatedPublicKey = (publicKey: string) =>
-  `${publicKey.slice(0, 4)}...${publicKey.slice(-4)}`;
+  `${publicKey.slice(0, 4)}…${publicKey.slice(-4)}`;
```

Comment:
Little stylistic thing here. It's better to have `“ ” ‘ ’ …` instead of `" " ' ' ...`